common.yml

```diff
@@ -20,6 +20,7 @@ mongo_tools_variables:
       - name: integration-4.0
       - name: integration-4.0-auth
       - name: integration-4.0-cluster
+      - name: integration-4.0-mmapv1
```

Comment:
Does this address https://jira.mongodb.org/browse/TOOLS-2446?

Reply:
Yes, this should have addressed TOOLS-2446